crits/standards/parsers.py

```diff
@@ -160,45 +253,179 @@ def parse_observables(self, observables):
+                                               user=analyst,
+                                               description=description,
+                                               title=title,
+                                               data_type="Text",
+                                               tool_name="STIX",
+                                               tool_version=None,
+                                               method=self.source_instance.method,
+                                               reference=self.source_instance.reference)
+                    self.parse_res(imp_type, obs, res)
+                elif (isinstance(item, File) and
+                      item.custom_properties and
+                      item.custom_properties[0].name == "crits_type" and
+                      item.custom_properties[0]._value == "Certificate"):
+                    imp_type = "Certificate"
+                    description = str(item.description)
+                    filename = str(item.file_name)
+                    data = None
+                    for obj in item.parent.related_objects:
+                        if isinstance(obj.properties, Artifact):
+                            data = base64.b64decode(obj.properties.data)
+                    res = handle_cert_file(filename,
+                                           data,
+                                           self.source,
+                                           user=analyst,
+                                           description=description)
+                    self.parse_res(imp_type, obs, res)
+                elif isinstance(item, File) and self.has_network_artifact(item):
+                    imp_type = "PCAP"
+                    description = str(item.description)
+                    filename = str(item.file_name)
+                    data = None
+                    for obj in item.parent.related_objects:
+                        if (isinstance(obj.properties, Artifact) and
+                            obj.properties.type_ == Artifact.TYPE_NETWORK):
+                            data = base64.b64decode(obj.properties.data)
+                    res = handle_pcap_file(filename,
+                                           data,
+                                           self.source,
+                                           user=analyst,
+                                           description=description)
+                    self.parse_res(imp_type, obs, res)
+                elif isinstance(item, File):
+                    imp_type = "Sample"
+                    filename = str(item.file_name)
+                    md5 = item.md5
+                    data = None
+                    for obj in item.parent.related_objects:
+                        if (isinstance(obj.properties, Artifact) and
+                            obj.properties.type_ == Artifact.TYPE_FILE):
+                            data = base64.b64decode(obj.properties.data)
```

Comment:
I wonder if it does the same things for PCAP, RawData, and Certificate binary data...